tests/test_bot.py

```diff
@@ -2084,6 +2083,15 @@ def test_send_message_default_parse_mode(self, default_bot, chat_id):
         assert message.text == test_markdown_string
         assert message.text_markdown == escape_markdown(test_markdown_string)
 
+    @flaky(3, 1)
+    @pytest.mark.parametrize('default_bot', [{'protect_content': True}], indirect=True)
+    def test_send_message_default_protect_content(self, default_bot, chat_id):
```

Comment:
for the other default settings that we *can* actually test, we do so for all the methods (search for `_default_parse_mode` in the tests). Having more tests is better than having less 😬

Reply:
wow, guess there's a lot of new tests to be written then

Reply:
the joys of development 🥳